docs/Jira-Project-Migrations.md

```diff
@@ -0,0 +1,105 @@
+
+## Create Empty Project In Target Jira Instance
+
+The Jira import mechanism requires the target Jira instance to have an existing project to serve as a receiver for the imported data.  The project must have the same project `Key` value as the source project being imported.
+
+Steps to create an empty project:
+*  Log into the web management console using an account with Jira administrator privileges.
+*  Select `Projects` on the top menu and then `Create project`.
+*  Select the project format that matches the source project and click `Next`.
+*  Select the default Issue Types and Workflow and click `Select`.
+*  Enter values for the `Name`, `Key`, select the `Project Lead`, and click `Submit`.  **IMPORTANT: The Key value must match the source project**.
+
+Verify the empty project was created by selecting `Projects` on the top menu and select `View All Projects`.  Search for the Name or Key that matches the newly created project.
+
+## Create Matching Configurations In Target Jira Instance
+
+After creating the empty project on the target Jira instance to receive the import, it needs to be configured to match the source project in order for the Jira import mechanism to complete properly.  It must use the same issue types, workflows, screens, fields, priorities, and any other custom configurations.  To view and compare the configurations, click on the gear icon in the top right and select `Issues` under `JIRA ADMINISTRATION`.  This will open the page where all project issue configurations can be modified.  Perform this procedure on both the source and target Jira instances and compare the settings for the empty target project and the source project.
+
+Below is a list of items to review and modify to match between the source and target projects:
+
+*  **ISSUE TYPES** - `Issue types`, `Issue type schemes`, `Sub-tasks`
+*  **WORKFLOWS** - `Workflows`, `Workflow schemes`
+(Note: A copy of the source project `Workflow` can be exported from the source project and then imported to the new target project. It is not necessary to re-create it from scratch.)
+*  **SCREENS** - `Screens`, `Screen schemes`, `Issue type screen schemes`
+*  **FIELDS** - `Custom fields`, `Field configurations`, `Field configuration schemes`
+*  **PRIORITIES, ISSUE FEATURES, ISSUE ATTRIBUTES** - Jira projects tend to use the defaults for these configurations but a quick review should be performed to confirm if any custom settings were created.
+
+## Create Users/Groups/Roles In Target Jira Instance
+
+Jira's project import mechanism will create local users attached to the imported project if it can not find a matching username on the target Jira instance.  If the desire is to re-link the users to an external directory (i.e. Active Directory) and/or maintain user links to the issues, the users must be pre-created in the target Jira instance before the import and the username must match exactly.
```

Comment:
Backend user _IDs_ don't matter, correct: just the logical user names within a shared name-space?

Reply:
Yes.  It's just the user names in the Jira namespace irrespective of the source (internal or external).

How's this updated verbiage:

```
Jira's project import mechanism will create local users attached to the imported project if it can not find a matching user name on the target Jira instance.  If the desire is to not create new users, maintain user names of the target Jira instance, and  maintain user name links to the issues on the source Jira instance, the user names on the source Jira instance must be changed to match exactly that of the target Jira instance.
```